demos/time_series_demo/python/time_series_forecasting_demo.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
```

Comment:
The name of the main file should match the name of the demo directory. Please rename one or the other.

Reply:
done